docs/app/services/plunker/templates/config_js.txt

```diff
@@ -28,6 +28,7 @@ System.config({
     '@angular/platform-browser': 'https://unpkg.com/@angular/platform-browser/bundles/platform-browser.umd.js',
     '@angular/platform-browser-dynamic': 'https://unpkg.com/@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
     '@angular/upgrade/static': 'https://unpkg.com/@angular/upgrade/bundles/upgrade-static.umd.js',
+    '@angular/animations': 'https://unpkg.com/@angular/animations/bundles/animations.umd.js',
```

Comment:
Need to add `@angular/animations/browser` and `@angular/platform-browser/animations` packages in here also. That will fix the delay in animations